tpm/structures.go

```diff
@@ -67,6 +72,18 @@ type pcrInfo struct {
 	DigestAtCreation digest
 }
 
+type nvDataPublic struct {
+	tag          tpmutil.Tag
+	index        tpmutil.Handle
+	pcrshort1    pcrInfoShort
+	pcrshort2    pcrInfoShort
+	Attrbute     nvAttribute
```

Comment:
Spelling: "Attrbute" -> "Attribute"

Also, why is this exported when the other fields are not? Should this be unexported to "attribute"?